### DISCUSSION TITLE
Requests

### DISCUSSION THREAD

**Discussion by Woshimbo** [community/community#49906] [Mobile]
A simpler way for mobile users to use GitHub by integrating text editors instead of your own. (There are plenty out there which, in case GitHub Mobile made a file provider location or simply act as a folder which would hold the repository that's chosen could act as an easy way to modify files with the various text editors.)

Trying to do work from nothing but the app and browser if needed.

<details>
<summary>ℹ️ Additional Debug Info</summary>
GitHub for iOS, Version 1.103.0 (93212509)
Device: iPhone 12 (iOS 16.3.1)
</details>

**Comment by aconsuegra** [comment]
Hi @RollingBytes, thanks for getting in touch! Are you finding any issue in particular with the existing file editor that we use in the GitHub iOS app?

**Comment by Woshimbo** [comment]
Not particularly, you know how we all have our little things to pick at though. The only thing I would love to see is somehow I could integrate GitHub more in to what I’m working on which is purely on Pythonista.It would be amazing to start interacting with the app from a customized work area, haha.In good faith.On Mar 22, 2023, at 10:17, Antonio J. Consuegra ***@***.***> wrote:﻿
Hi @RollingBytes, thanks for getting in touch! Are you finding any issue in particular with the existing file editor that we use in the GitHub iOS app?
